core/src/main/scala/io/finch/request/package.scala

```diff
@@ -309,7 +358,9 @@ package object request extends LowPriorityImplicits {
    *
    * @param message the message
    */
-  class RequestReaderError(val message: String) extends Exception(message)
+  class RequestReaderError(val message: String, cause: Throwable) extends Exception(message, cause) {
+    def this(message: String) = this(message, null)
```

Comment:
We can add a comment, "interfacing with Java here", so that Scala devs do not get a heart attack when they see the null. :-)